demos/global-matrix-auth/README.md

```diff
@@ -19,6 +19,27 @@ jenkins:
   authorizationStrategy:
     projectMatrix:
       grantedPermissions:
+        - "View/Delete:authenticated"
+        - "View/Read:authenticated"
+        - "View/Configure:authenticated"
+        - "View/Create:authenticated"
+        - "Job/Read:authenticated"
+        - "Job/Build:authenticated"
+        - "Job/Configure:authenticated"
+        - "Job/Create:authenticated"
+        - "Job/Delete:authenticated"
+        - "Job/Discover:authenticated"
+        - "Job/Move:authenticated"
+        - "Job/Workspace:authenticated"
+        - "Job/Cancel:authenticated"
+        - "Run/Delete:authenticated"
+        - "Run/Replay:authenticated"
+        - "Run/Update:authenticated"
+        - "SCM/Tag:authenticated"
         - "Overall/Read:anonymous"
         - "Overall/Administer:authenticated"
 ```
+
+Some permissions depends on actual plugin-usage. 
+For Example: "Release/*:authenticated" is only available if you _use_ the Releaseplugin in one of your jobs. 
```

Comment:
add another newline above as suggested by codacy :smile:
The example is not that readable when viewing it.

https://github.com/jenkinsci/configuration-as-code-plugin/blob/e6ff1680bcd11a61617a70606be89da13c95832a/demos/global-matrix-auth/README.md

Reply:
```suggestion
For Example: "Release/*:authenticated" is only available if you _use_ the Release plugin in one of your jobs. 
```